config/vufind/BrowZine.ini

```diff
@@ -68,6 +73,26 @@ fullTextFile = "PDF Full Text|browzine-pdf|https://assets.thirdiron.com/images/i
 retractionNoticeUrl = "View Retraction Notice|browzine-retraction"
 expressionOfConcernNoticeUrl = "View Expression of Concern Notice|browzine-concern"
 
+; The bestIntegratorLink service is not yet documented on that wiki page, but it provides
+; "the best link available" according to Third Iron's algorithm. See
+; https://support.thirdiron.com/support/solutions/articles/72000569332-open-access-linking-and-configuration-options).
+; The label and icon values specified here are defaults, but are overridden based on the
+; type of "best" link actually provided for a given title. See [BestIntegratorLinks] below. 
+;bestIntegratorLink = "Get full text|browzine-best"
+
+; When 'bestIntegratorLink' is enabled in DOIServices, this section defines what label and
+; icon to display based on the type of "best" link, overriding defaults in BrowzineFactory.
+; If this section is enabled but an individual link type encountered is not, the label and
+; icon will come from 'bestIntegratorLink' above.
+; See also useBrowzineLabel within [IdentifierLinks] above.
+;[BestIntegratorLinks]
+;fullTextFile = "PDF Full Text|browzine-pdf|https://assets.thirdiron.com/images/integrations/browzine-pdf-download-icon.svg"
+;retractionNoticeUrl = "View Retraction Notice|browzine-retraction"
+;expressionOfConcernNoticeUrl = "View Expression of Concern Notice|browzine-concern"
+;contentLocation = "view_article|browzine-view"
```

Comment:
Per the docs, `contentLocation` means "A link to the landing page for this article", and the `recommendedLinkText` is the appropriately generic "View Article".  I don't see anything in en.ini that really captures that idea except "View Record", which implies within VuFind itself and I think would be confusing.

Reply:
Agreed -- it's changed to `Get full text`.